package.json

```diff
@@ -56,7 +56,7 @@
     "clean:nuxt": "shx rm -rf \"**/.nuxt\"",
     "clean:dist": "shx rm -rf \"**/dist\"",
     "clean:turbo": "shx rm -rf \"**/.turbo\"",
-    "dev": "turbo dev --concurrency=100% --filter './examples/*' --filter './integrations/*' --filter=@scalar/nuxt --filter=@scalar/draggable --filter @scalar/components --filter @scalar/api-client --filter @scalar/nextjs-openapi",
+    "dev": "turbo dev --concurrency=20 --filter './examples/*' --filter './integrations/*' --filter=@scalar/nuxt --filter=@scalar/draggable --filter @scalar/components --filter @scalar/api-client --filter @scalar/nextjs-openapi",
```

Comment:
the issue with 20 is it limits to 20 threads, what was wrong with 100%?

Reply:
I think we had 20 before and it was breaking on machines that did not have 20 threads. Also 100% allows for running more apps

Reply:
@hanspagel we should try to move these kinds of changes into another pr, cause the fix you did works! but this is a separate issue and we cant merge smth that isnt atomic.

Reply:
I wrote it in the PR description:

> I’ve also had to increase the concurrency for pnpm dev, because I don’t have enough cores to run that many dev servers in parallel.

`20` working fine if you have less than 20 cores, but `100%` is not working if you have less cores than scripts you try to run.

Sooo… without that it’s not working on my machine, so it’s OK to have it in the same PR I’d say.

Reply:
ooooo really, I thought 100% was 100% of your threads, not 100% of the scripts
> Use 100% to use all available logical processors.

https://turborepo.com/docs/reference/run#--concurrency-number--percentage

Reply:
either way lets boot it to another PR?